examples/src/routes/VisualizationByUri.jsx

```diff
@@ -0,0 +1,100 @@
+import VisualizationTreemapExample from '../components/VisualizationTreemapByUriExample';
+import VisualizationHeatmapExample from '../components/VisualizationHeatmapByUriExample';
+
+import VisualizationColumnChartByUriExampleSRC from '!raw-loader!../components/VisualizationColumnChartByUriExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationTableExampleSRC from '!raw-loader!../components/VisualizationTableByUriExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationBarExampleSRC from '!raw-loader!../components/VisualizationBarByUriExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationLineExampleSRC from '!raw-loader!../components/VisualizationLineByUriExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationAreaExampleSRC from '!raw-loader!../components/VisualizationAreaByUriExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationHeadlineExampleSRC from '!raw-loader!../components/VisualizationHeadlineByUriExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationScatterExampleSRC from '!raw-loader!../components/VisualizationScatterByUriExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationBubbleExampleSRC from '!raw-loader!../components/VisualizationBubbleByUriExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationPieExampleSRC from '!raw-loader!../components/VisualizationPieByUriExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationDonutExampleSRC from '!raw-loader!../components/VisualizationDonutByUriExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationTreemapExampleSRC from '!raw-loader!../components/VisualizationTreemapByUriExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+import VisualizationHeatmapExampleSRC from '!raw-loader!../components/VisualizationHeatmapByUriExample'; // eslint-disable-line import/no-webpack-loader-syntax, import/no-unresolved, import/extensions, import/first
+
+export const VisualizationByUri = () => (
+    <div>
+        <h1>Visualization</h1>
+
+        <p>These are examples of the generic Visualization component.</p>
```

Comment:
I would specify that this test is about URI.
`These are the examples of the generic Visualization component that uses URI to identify insights.`